compiler/src/iree/compiler/Codegen/Common/LinkTuningSpecsPass.cpp

```diff
@@ -85,6 +85,14 @@ emitLinkedTuningSpec(ModuleOp module, ArrayRef<NamedSequenceOp> specsToLink) {
   newSpec->setAttr(kTuningSpecEntrypointAttrName, builder.getUnitAttr());
   module->setAttr(kTuningSpecDefaultEntrypointAttrName, builder.getUnitAttr());
 
+  for (auto innerModule : module.getBody()->getOps<ModuleOp>()) {
+    // Remove the default tuning spec attribute from inner modules,
+    // as the top-level module is attached with default attribute.
```

Comment:
Do we remove it because the entrypoint name can change? If that's the case, I think we'd want to iterate over the parents of `specsToLink` instead, since this functions doesn't assume a specific nesting structure.